placeTypes.ts

```diff
@@ -0,0 +1,49 @@
+import type { ObjectId } from "mongodb";
+import type { Place as GooglePlace } from "@googlemaps/google-maps-services-js";
+
+export type PlaceWithAccesibilityData = GooglePlace & {
```

Comment:
Is there a reason why the promotion fields are not included in this type?

Reply:
Thats actually a good point. I will add those fields then!